src/v/kafka/protocol/types.h

```diff
@@ -136,4 +136,20 @@ concept KafkaApi = requires(T request) {
     { T::min_flexible } -> std::convertible_to<const api_version&>;
 };
 
+/*
+ * Data type of the configuration entry.
+ */
+enum class describe_configs_type : int8_t {
+    unknown = 0,
+    boolean = 1,
+    string = 2,
+    _int = 3,
+    _short = 4,
+    _long = 5,
```

Comment:
Do some research to see what the integral types in Java are. My type map should be the same.

Reply:
>Do some research to see what the integral types in Java are. My type map should be the same.

https://github.com/apache/kafka/blob/be032735b39360df1a6de1a7feea8b4336e5bcc0/core/src/main/scala/kafka/server/ConfigHelper.scala#L193-L207

I recommend installing something like jetbrains idea to navigate kafka source code it makes things pretty nice.

Reply:
Thanks! Although my comment was about researching type sizes in Java instead of names